client/wfnews-war/src/main/angular/src/app/services/report-of-fire-service.ts

```diff
@@ -0,0 +1,31 @@
+import { HttpClient, HttpRequest } from "@angular/common/http";
+import { Injectable } from "@angular/core";
+import { AppConfigService } from "@wf1/core-ui";
+import { Observable } from "rxjs/internal/Observable";
+
+@Injectable({
+    providedIn: 'root'
+})
+export class ReportOfFireService {
+    rofUrl: string;
+
+    constructor(private appConfigService: AppConfigService, private httpClient: HttpClient) {  }
+
```

Comment:
`any` is OK, but it's better to use defined types. Images should have a type (Blob or whatever). For the reportOfFire, you have a defined const above that could just as easily be turned into a model object to make it explicit.

`type ReportOfFire = { ... }`